packages/wonder-blocks-modal/components/modal-content-pane.js

```diff
@@ -38,6 +39,7 @@ type Props = {
 export default class ModalContentPane extends React.Component<Props> {
```

Comment:
I was a little confused when I saw this again b/c I thought there would just content within this component.  Maybe `ModalPanel` would be better.